Sources/SourceKitLSP/Swift/SwiftTestingScanner.swift

```diff
@@ -318,18 +332,28 @@ final class SyntacticSwiftTestingTestScanner: SyntaxVisitor {
       return .skipChildren
     }
 
-    let parameters = node.signature.parameterClause.parameters.map {
-      "\($0.firstName.identifier?.name ?? $0.firstName.text):"
+    let parameters = node.signature.parameterClause.parameters.map { param in
+      let result =
+        if let identifier = param.firstName.identifier {
+          backtickIfNeeded(identifier).name
+        } else {
+          // Something like `_`, leave as-is.
+          param.firstName.text
+        }
+      return "\(result):"
     }.joined()
-    let name = "\(identifier.name)(\(parameters))"
+
+    let (hasBackticks, baseName) = backtickIfNeeded(identifier)
+    let fullName = "\(baseName)(\(parameters))"
+    let displayName = attributeData.displayName ?? (hasBackticks ? identifier.name : fullName)
```

Comment:
Does this mean that we include the parameter labels only if the base name doesn’t require backticks? I don’t see the relation between omitting parameter labels and a backticked base name but maybe I’m missing something.

Reply:
OK, interesting choice and I think I would have treated raw identifiers just like any other identifier (expecting that wrapping an identifier in backticks wouldn't cause a real change) but I don't have strong enough of an opinion to argue about it. 

Could you add a short comment with your explanation though since the choice isn't totally obvious.

Reply:
Added a comment in https://github.com/swiftlang/sourcekit-lsp/pull/2191

Reply:
Thank you 😄